### PR TITLE
Fix `total_counts` to `total_count` in the module_user_guide

### LINKED ISSUE #39: module_user_guide.ipynb: NegativeBinomial function call raises "unexpected keyword argument 'total_counts'" error
I am trying to create my own PyTorch module by looking at the documentation as in [Crafting the Module in vanilla PyTorch](https://docs.scvi-tools.org/en/stable/tutorials/notebooks/module_user_guide.html#Crafting-the-Module-in-vanilla-PyTorch). In the next step, I want to implement the model as in [Constructing a high-level model](https://docs.scvi-tools.org/en/stable/tutorials/notebooks/model_user_guide.html). 

In the first linked page, the `loss` function has a line, in cell 6, 
`log_lik = NegativeBinomial(total_counts=theta, total=nb_logits).log_prob(x).sum(dim=-1)`

whereas the [PyTorch documentation](https://pytorch.org/docs/1.2.0/distributions.html#torch.distributions.negative_binomial.NegativeBinomial) for the function is given as
`torch.distributions.negative_binomial.NegativeBinomial(total_count, probs=None, logits=None, validate_args=None)` 

So, in the next stage, when I am replacing the built-in `VAE` model with `MyModule`, I am getting the `unexpected keyword argument 'total_counts'` error.

I am attaching the file for reproducing the issue. 

[scvi_tut.py.txt](https://github.com/YosefLab/scvi-tutorials/files/7789894/scvi_tut.py.txt)

I am using PyTorch 1.10.0+cpu.

### PR DESCRIPTION
**Pull request by watiss**
Fixes #39

### CODE REVIEW

**Review by rangan2510 — CHANGES_REQUESTED** [review]
This does not work as now, "total" is an unexpected keyword according to the PyTorch documentation for `NegativeBinomial`. Have a look at the attached file and try to run a user module for clustering. 
[scvi_tut.py.txt](https://github.com/YosefLab/scvi-tutorials/files/7791476/scvi_tut.py.txt)
